index.html

```diff
@@ -129,7 +129,7 @@ <h2 class="hone">Facebook 360</h2>
       <div class="table">
         <h2 class="hone">Uber Navigation</h2>
```

Comment:
As the previous reviewer requested, please generate all works cards dynamically. You should remove this section on HTML and generate it dynamically.